### DISCUSSION TITLE
i cant start 3d texturized with dream texture on blender

### DISCUSSION THREAD

**Discussion by nil380** [carson-katri/dream-textures#570] [Q&A]
stabilityai/stable diffusion 2 depht give a message of error he dont foud principled BSDF key
![2023-02-14 12_09_37-Blender](https://user-images.githubusercontent.com/125371266/219353807-1b636fc4-2deb-4c19-a3b0-35b9adc75f60.png)

**Comment by KrashlTest** [comment]
Hi! I had the same mistake. After I switched the Blender language to English, everything worked.

**Comment by better319** [comment]
我这个是成功了，但是，这个材质怎么搞的一点都不像
